packages/mdc-button/_mixins.scss

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/ripple/mixins";
+@import "@material/theme/mixins";
+@import "@material/theme/variables";
+
+@mixin mdc-button-filled() {
+  @include mdc-ripple-base;
+  @include mdc-ripple-bg((pseudo: "::before", base-color: white, opacity: .32));
+  @include mdc-ripple-fg((pseudo: "::after", base-color: white, opacity: .32));
+  @include mdc-theme-prop(color, text-primary-on-dark);
+
+  background-color: black;
+
+  fieldset:disabled &,
```

Comment:
Please test the demo after merging master into this branch again, and ensure this doesn't re-regress #1167.